plugin-hrm-form/src/channels/setUpChannels.tsx

```diff
@@ -121,35 +123,35 @@ export const setupDefaultChannels = () => {
   setupDefaultChannel();
 };
 
-export const setupTwitterChatChannel = () => {
-  const TwitterChatChannel = DefaultTaskChannels.createChatTaskChannel(
-    'twitter',
-    task => task.channelType === 'twitter',
+export const setupTelegramChatChannel = () => {
+  const TelegramChatChannel = DefaultTaskChannels.createChatTaskChannel(
+    'telegram',
+    task => task.channelType === 'telegram',
```

Comment:
```suggestion
    task => task.channelType === 'telegram' || task.attributes.customChannelType === 'telegram',
```

This check needs to be equivalent to the line version on line185 of this file - it needs to check `task.attributes.customChannelType` as well to work for conversations. The channelType property gets overwritten in conversations so we must use an alternative for this purpose

Reply:
@stephenhand - This is done. Thank you. I have also added the icon svg from Jana.